src/main/java/address/sync/task/CloudUpdateTask.java

```diff
@@ -28,13 +28,13 @@ public void run() {
 
         List<UniqueData> allData = new ArrayList<>();
         allData.addAll(personsData);
-        allData.addAll(groupsData);
+        allData.addAll(tagsData);
 
         File mirrorFile = PrefsManager.getInstance().getMirrorLocation();
-        try {
-            simulator.requestChangesToCloud(mirrorFile, personsData, groupsData);
-        } catch (JAXBException e) {
-            System.out.println("Error requesting changes to the cloud");
-        }
+//        try {
```

Comment:
Should add in a comment to explain why this is temporarily disabled:
`The way the local model should use the API to push its update is not yet finalized.`